### DISCUSSION TITLE
How can I change(reconnect) forked repository?

### DISCUSSION THREAD

**Discussion by sorealk** [community/community#28723] [Other Feature Feedback, Questions, & Ideas]
Hi, this is my situation.

I forked original(called A) repository. 
but oneday my repository(called B) is connected to other forked repository(called C)
I want to reconnect my repository(B) to original repository(A).
How can I change the connection?

I don't know why It happend and how I fix it. plz help me.

Thank you.

**Comment by danielmeppiel** [comment]
Hi @sorealk ,

Let me see if I understood correctly:

- You created a fork of repo A
- When you access this fork on GitHub, the "forked from" tagline under the repo title points to repo C instead of repo A

This is the tagline I refer to (just an example of one of my public forks):

<img width="582" alt="image" src="https://user-images.githubusercontent.com/51440732/183419861-cc314e95-ff40-485d-b68f-9091f3e57bc0.png">

If that is the case, it may be that the upstream repo (called A) changed name, without any but a cosmetic impact on your fork. 

Let me know if I didn't get something right!

**Comment by Rudd-O** [comment]
We have a similar situation.

I forked a project from @kozec, and then my fork got forked into an organization @syncthing-gtk.

We would like to change the "fork base" of the @syncthing-gtk project back to use @kozec's repo as a base.

How can we make this change?

Thanks.
